src-electron/db/query-impexp.js

```diff
@@ -674,8 +677,24 @@ WHERE
   } else if (reportingPolicy == dbEnums.reportingPolicy.prohibited) {
     attribute.reportable = false
   }
-
-  if (storagePolicy == dbEnums.storagePolicy.attributeAccessInterface) {
+  if (attributeId) {
+    forcedExternal = await queryUpgrade.getForcedExternalStorage(
+      db,
+      attributeId
+    )
+    storageOption = await queryUpgrade.computeStorage(
+      db,
+      cluster.name,
+      null,
```

Comment:
Why is this null instead of the cluster ref?

Reply:
its not needed; why retrieve it and pass it if it won't be used? clusterRef is needed in other scenarios but when it is not needed there is not point of retrieving it

Reply:
will separate into two functions